.eslintrc.json

```diff
@@ -2,19 +2,31 @@
   "root": true,
   "extends": [
     "eslint:recommended",
-    "plugin:node/recommended",
+    "plugin:n/recommended",
     "plugin:import/recommended",
     "next/core-web-vitals",
     "plugin:prettier/recommended"
   ],
   "rules": {
     // Without 'modules' ignored would get 'Import and export declarations are
     // not supported yet' error in all ts files
+    /*
     "node/no-unsupported-features/es-syntax": [
       "error",
       { "ignores": ["modules"] }
     ],
+    */
 
+    // didn't get this working for now. I tried to allow "styles/globals.css" and make everything
+    // else throw an error. Added a couple eslint-disable-next-line lines to fix the problem for now 
+
+
+    /*
+    "n/no-extraneous-import": ["error", {
+      "allowModules": ["@styles/globals.css"]
+    }],
+    */ 
+    
```

Comment:
Vaihdoin `eslint-plugin-node`n käyttämään `eslint-plugin-n`:ää. `node`a ei päivitetä enää.

Heitti seuraavaa virhettä: `TypeError: context.getScope is not a function`

ESLint itse ehdotti seuraavia muutoksia: https://eslint.org/docs/latest/use/troubleshooting/v9-rule-api-changes mutta helpompi asentaa `eslint-plugin-n`, joka näyttäisi korjaavan ongelmat

En saanut `n/no-extraneous-import`-sääntöä toimimaan niin, että "/styles/global.css" olisi hyväksytty. Lisäsin manuaaliset ohitus kommenttirivin, jotta "ongelmasta" päästiin ohi :sweat_smile: 

```ts
// eslint-disable-next-line n/no-extraneous-import
import '~/styles/globals.css';
```

Reply:
Kommentoin vanhan tavan, olisin voinut periaattessa poistaa suoraankin. Poistan ne kun PR on ok / seuraavassa commitissa